jobs/build/release-schedules/Jenkinsfile

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env groovy
+
+node {
+    try {
+        checkout scm
+        commonlib = load("pipeline-scripts/commonlib.groovy")
+        slacklib = commonlib.slacklib
+        
+
+        stage("install the schedule-generator tool") {
+            def install = [
```

Comment:
Unfortunately, due to security constrains, we can't build and execute from a repo that's not controlled by ART. I will statically build the code and install it on the Jenkins host. Just assume `schedule-generator` will be in $PATH (I'll need to bump it as changes are made). I'll make sure to file cards to migrate these workloads to less constrained hosts so it can be more dynamic.